seacrowd/sea_datasets/mswc/mswc.py

```diff
@@ -0,0 +1,308 @@
+                ],
+                "opus": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/cnh_opus/validation/0000.parquet?download=true",
+                ],
+            },
+            "ind": {
+                "wav": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/id_wav/validation/0000.parquet?download=true",
+                ],
+                "opus": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/id_opus/validation/0000.parquet?download=true",
+                ],
+            },
+            "vie": {
+                "wav": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/vi_wav/validation/0000.parquet?download=true",
+                ],
+                "opus": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/vi_opus/validation/0000.parquet?download=true",
+                ],
+            },
+        },
+        "test": {
+            "cnh": {
+                "wav": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/cnh_wav/test/0000.parquet?download=true",
+                ],
+                "opus": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/cnh_opus/test/0000.parquet?download=true",
+                ],
+            },
+            "ind": {
+                "wav": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/id_wav/test/0000.parquet?download=true",
+                ],
+                "opus": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/id_opus/test/0000.parquet?download=true",
+                ],
+            },
+            "vie": {
+                "wav": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/vi_wav/test/0000.parquet?download=true",
+                ],
+                "opus": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/vi_opus/test/0000.parquet?download=true",
+                ],
+            },
+        },
+    },
+}
+
+_SUPPORTED_TASKS = [Tasks.SPEECH_RECOGNITION]
+_SUPPORTED_SCHEMA_STRINGS = [f"seacrowd_{str(TASK_TO_SCHEMA[task]).lower()}" for task in _SUPPORTED_TASKS]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+@dataclass
+class SMSASeacrowdConfig(SEACrowdConfig):
+    """BuilderConfig for Nusantara."""
+
+    language: str = None
+    audio_format: str = None
```

Comment:
I'd say it's better to remove dependency between necessary data for the logic and the subset name, assuming we might change the subset name format and we want to avoid changing the existing logic to retrieve language and audio_format. What do you think? @sabilmakbar

Reply:
Hmm the subset name should be consistent over time (hence guaranteeing the code consistency over time) and the the way we construct it (by concatting `lang_name` and `format_name` in `L88` and `L106`) is acceptable by both reviewers, so I don't think this is necessary.

Also, the config name should be prepended by `_DATASETNAME` to follow other dataloader implementations.